src/Microsoft.ML.Core/Utilities/DoubleParser.cs

```diff
@@ -70,25 +70,26 @@ public enum Result
             Error = 3
         }
 
-        public static Result Parse(out Single value, string s, int ichMin, int ichLim)
+        public static Result Parse(out Single value, ReadOnlyMemory<char> s)
```

Comment:
These methods would be more usable/applicable if they took `ReadOnlySpan<char>` instead. That way if someone only has a Span, they can still call this function.

Reply:
So true.

---
In reply to: [216471029](https://github.com/dotnet/machinelearning/pull/863#discussion_r216471029) [](ancestors = 216471029)